src/torrent.ts

```diff
@@ -112,7 +112,9 @@ export function saveTorrentFile(
 	const { outputDir } = getRuntimeConfig();
 	const buf = meta.encode();
 	const name = stripExtension(meta.name);
```

Comment:
will this `stripExtension` bit still work?

Reply:
You mean if the name has a / in it?

I don't think it will strip it....considering it wont endWith just the extension. But considering that if it ends with a / it wont have an extension, I don't think we would technically really need to worry about that? It's not a single file torrent.

Thoughts?

Reply:
makes sense